src/test/java/com/karankumar/booksapi/repository/BookRepositoryTest.java

```diff
@@ -74,13 +72,31 @@ void findSavedBooks() {
             softly.assertThat(result).containsExactlyInAnyOrder(book);
         });
     }
+
+    @Test
+    @DisplayName("find book with isbn")
+    void findBookByIsbn() {
+        createAndSaveAuthors();
+        Book book = createBookwithISBN();
+        bookRepository.save(book);
+
+        Book result = bookRepository.findBookByISBN("978-3-16-148410-0");
+        Assertions.assertEquals(book,result);
```

Comment:
As per our style guide, we're using AssertJ assertions, not JUnit. Could you please change this?